### PR TITLE
docs(form): Add KB / Improve FormItemsTemplate and UI refreshing information

### PR DESCRIPTION
**Pull request by dimodi**

### CODE REVIEW

File: components/form/formitems/overview.md [hunk 1]
```diff
@@ -15,8 +15,8 @@ You can customize the [default editors]({%slug form-overview%}#automatic-generat
 In this article:
 
 * [FormItem Parameters](#formitem-parameters)
-* [Example](#example)
 * [Add Form Fields to Autogenerated Ones](#add-form-fields-to-autogenerated-ones)
+* [UI rendering and performance optimization inside the Form](#ui-rendering-inside-the-form)
```
Comment (yordan-mitev) [inline components/form/formitems/overview.md:19]: Use a consistent capitalization style in the list.

File: knowledge-base/form-use-items-in-custom-component.md [hunk 2]
```diff
@@ -0,0 +1,206 @@
...
+1. Define [group renderers]({%slug form-formitems-formitemstemplate%}#form-group-renderer) and [item renderers]({%slug form-formitems-formitemstemplate%}#form-item-renderer) for the Form groups and items, which will be children of the `<FormItemsTemplate>`.
+1. Add the custom Razor components inside `<FormItemsTemplate>`.
+1. Define parameters for the custom Razor components, which will receive Form groups or Form items.
+1. Define the required renderers inside the custom Razor components, according to the parameters from the previous step.
+
+### Example
+
+The example below assumes that the project name is `YourAppName` and the `Person` class is defined in namespace `YourAppName.Data`. Rename the namespaces in `Home.razor`, `MultipleFormItems.razor`, and `Person.cs` before running the code in your app.
+
+* `Home.razor` is the main Razor file, which holds the Form
```
Comment (yordan-mitev) [inline knowledge-base/form-use-items-in-custom-component.md:50]:
```suggestion
* `Home.razor` is the main Razor file, which holds the Form.
```

**Review by yordan-mitev — APPROVED** [review]

File: components/form/formitems/overview.md [hunk 3]
```diff
@@ -15,8 +15,8 @@ You can customize the [default editors]({%slug form-overview%}#automatic-generat
 In this article:
 
 * [FormItem Parameters](#formitem-parameters)
```
Comment (dimodi) [inline components/form/formitems/overview.md:17]:
```suggestion
* [FormItem parameters](#formitem-parameters)
```

File: components/form/formitems/overview.md [hunk 4]
```diff
@@ -15,8 +15,8 @@ You can customize the [default editors]({%slug form-overview%}#automatic-generat
...
-* [Example](#example)
 * [Add Form Fields to Autogenerated Ones](#add-form-fields-to-autogenerated-ones)
```
Comment (dimodi) [inline components/form/formitems/overview.md:18]:
```suggestion
* [Combine explicit and autogenerated Form fields](#add-form-fields-to-autogenerated-ones)
```

**Pull request merged**